src/components/v2/ProgressBarHorizontal/styles.ts

```diff
@@ -5,10 +5,11 @@ export const useStyles = ({ over }: { over: boolean }) => {
   const theme = useTheme();
   return {
     slider: css`
+      display: block;
       color: ${over ? theme.palette.interactive.error50 : theme.palette.interactive.success};
       background-color: ${theme.palette.background.default};
       height: ${theme.spacing(2)};
-      padding: 0;
+      padding: 0 !important;
```

Comment:
Is there any way to avoid using important?

Reply:
Nope, MUI's CSS is horrible and uses a weird rule based on the cursor --'